### ISSUE TITLE
Record all face of cubemap projection

### ISSUE THREAD

**Issue by phunm211** [xmar/360Transformations#16]
When I set value of "useTile" is true in CubeMapCompact type, I've received only one video of front face. How can I get the other face?

**Comment by xmar** [comment]
Hi,
The useTile option encode the different faces into different video tracks inside the same mkv container. Here we do not encode the video with the hevc tile. If you look inside your mkv output file you should observe 6 video tracks.

**Comment by phunm211** [comment]
Yep I've seen that but I can't use this output file as a CubeMapCompact video to calculate its MS-SSIM point.
So if I set layoutFlow is 

> "layoutFlow= [["../input_equi.mp4", "Equirectangular", "FlatFixed"], ["../input_equi.mp4", "Equirectangular", "CubeMapCompact", "FlatFixed"]]"

and I set "useTile = true" then I change the ratio of bitrateFront,bitrateBack,etc.. .
Is my solution correct?
Thanks

**Comment by xmar** [comment]
Hello,

In fact if you define the flow ["../input_equi.mp4", "Equirectangular", "CubeMapCompact", "FlatFixed"], the cubeMap video with the « tiles » will never be encoded and stored on the disk. Only a projection from Equirectangular to CubeMapCompact will happen. If you want to compress the video with different bitrate budget per tile then you need first to run this flow ["../input_equi.mp4", "Equirectangular", "CubeMapCompact"]. This will generate a video file with the « tiled » cubemap projection encoded inside. Then you should compute the MS-SSIM by using this flow: [["../input_equi.mp4", "Equirectangular", "FlatFixed"], ["../yourCubeMapOutputFile", "CubeMapCompact", "FlatFixed"]].

**Comment by phunm211** [comment]
I've followed your instruction and Terminal get an error: "Segmentation fault (core dumped)". So how can I solve that?
Thanks.
